docs/build

```diff
@@ -388,6 +390,232 @@ def check_gcp_guides():
+    "colored_log_format",
+    "cookie_samesite",
+    "cookie_secure",
+    "dag_cleanup_interval",
+    "dag_default_view",
+    "dag_dir_list_interval",
+    "dag_discovery_safe_mode",
+    "dag_file_processor_timeout",
+    "dag_orientation",
+    "dag_processor_manager_log_location",
+    "dagbag_import_timeout",
+    "dags_are_paused_at_creation",
+    "dags_in_image",
+    "dags_volume_claim",
+    "dags_volume_host",
+    "dags_volume_mount_point",
+    "dags_volume_subpath",
+    "data_profiler_filter",
+    "default_cpus",
+    "default_disk",
+    "default_email_on_failure",
+    "default_email_on_retry",
+    "default_gpus",
+    "default_hive_mapred_queue",
+    "default_owner",
+    "default_ram",
+    "default_task_retries",
+    "default_wrap",
+    "delete_option_kwargs",
+    "delete_worker_pods",
+    "delete_worker_pods_on_failure",
+    "demo_mode",
+    "donot_pickle",
+    "enable_xcom_pickling",
+    "env_from_configmap_ref",
+    "env_from_secret_ref",
+    "error_logfile",
+    "expose_config",
+    "expose_hostname",
+    "expose_stacktrace",
+    "fab_logging_level",
+    "flower_host",
+    "flower_port",
+    "force_log_out_after",
+    "frontend",
+    "fs_group",
+    "git_branch",
+    "git_dags_folder_mount_point",
+    "git_repo",
+    "git_ssh_key_secret_name",
+    "git_ssh_known_hosts_configmap_name",
+    "git_subpath",
+    "git_sync_container_repository",
+    "git_sync_container_tag",
+    "git_sync_credentials_secret",
+    "git_sync_depth",
+    "git_sync_dest",
+    "git_sync_init_container_name",
+    "git_sync_rev",
+    "git_sync_root",
+    "git_sync_run_as_user",
+    "git_user",
+    "group_member_attr",
+    "hide_paused_dags_by_default",
+    "hide_sensitive_variable_fields",
+    "hostname_callable",
+    "ignore_malformed_schema",
+    "image_pull_secrets",
+    "job_heartbeat_sec",
+    "killed_task_cleanup_time",
+    "kinit_path",
+    "kube_client_request_args",
+    "load_default_connections",
+    "load_examples",
+    "log_animation_speed",
+    "log_auto_tailing_offset",
+    "log_fetch_delay_sec",
+    "log_fetch_timeout_sec",
+    "log_filename_template",
+    "log_format",
+    "log_processor_filename_template",
+    "logging_level",
+    "logs_volume_claim",
+    "logs_volume_host",
+    "logs_volume_subpath",
+    "mapred_job_name_template",
+    "max_num_rendered_ti_fields_per_task",
+    "max_tis_per_query",
+    "min_file_process_interval",
+    "navbar_color",
+    "num_runs",
+    "operation_timeout",
+    "page_size",
+    "plugins_folder",
+    "pod_template_file",
+    "print_stats_interval",
+    "proxy_fix_x_port",
+    "proxy_fix_x_prefix",
+    "proxy_fix_x_proto",
+    "sasl_enabled",
+    "scheduler_zombie_task_threshold",
+    "search_scope",
+    "secret_key",
+    "session_lifetime_days",
+    "simple_log_format",
+    "smtp_host",
+    "smtp_mail_from",
+    "smtp_port",
+    "smtp_ssl",
+    "smtp_starttls",
+    "smtp_user",
+    "sql_alchemy_connect_args",
```

Comment:
We already have docs: https://airflow.readthedocs.io/en/latest/configurations-ref.html#sql-alchemy-connect-args

Reply:
I think we should also have documentation in /docs/directory. Reference documentation (/airflow/config_tempates/config.yaml file) is important, but it does not easily find this feature by new users.  I would be happy if users would also add more detailed descriptions.

Here is an example of a good change.
https://github.com/apache/airflow/pull/9397/files
An option has been added to the reference catalog, but new documentation has also been added to the security.rst file.

It is also worth looking at how it is done by other projects. Kubernetes generates reference documentation.
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#ephemeralcontainer-v1-core
and also has normal documentation that is easier to understand and provides a broader description.  On the other hand, it is less accurate in some places.
https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/

Reply:
Oh yes, +1, I thought the script was still looking at the generated docs at configurations-ref, hence my comment